website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -61,7 +61,7 @@ the traffic redirection command is automatically set up via an init container.
 
 * To use transparent proxy on Kubernetes, Consul-helm >= `0.32.0` and Consul-k8s >= `0.26.0` are required in addition to Consul >= `1.10.0`.
 * If the default policy for ACLs is "deny", then Service Intentions should be set up to allow intended services to connect to each other.
```

Comment:
```suggestion
* If your default ACLs policy is `deny`, you must configure service intentions to explicitly allow services to connect to each other.
```
Use lower case for service intentions because it is a regular noun.
Use active voice.